requirements/base.in

```diff
@@ -28,6 +28,7 @@ pyproj
 pyshp
 rapidfuzz
 requests>=2.22.0
+httpx==0.28.1
```

Comment:
We are taking on a special HTTP client and mocking library for the async calls, but I think the performance benefit of doing the requests in parallel is worthwhile.